bin/configs/java-okhttp-gson-oneOf.yaml

```diff
@@ -0,0 +1,7 @@
+generatorName: java
+outputDir: samples/client/echo_api/java/okhttp-gson-oneOf
```

Comment:
what about putting it in `samples/client/others/java/okhttp-gson-oneOf` instead as the spec has nothing to do with echo api?

Also can you please add the new sample folder to `.github/workflows/samples-java-client-jdk11.yaml` as well?

Reply:
Both things are changed :)